<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScDialog</class>
 <widget class="QDialog" name="ScDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>扫码对话框</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接收到新的扫描码，请核对或修改后确认！</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>220</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btexit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btlogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确定投入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前扫描码为:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbsc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编辑或修改扫描码:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbInput1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator::unchecked { 
 image: url(:/images/close.png);
}
QCheckBox::indicator::checked { 
 image: url(:/images/open.png);
}
QCheckBox::indicator { 
 width: 30px;
 height: 30px;
}</string>
   </property>
   <property name="text">
    <string>人检1投入</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbInput2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator::unchecked { 
 image: url(:/images/close.png);
}
QCheckBox::indicator::checked { 
 image: url(:/images/open.png);
}
QCheckBox::indicator { 
 width: 30px;
 height: 30px;
}</string>
   </property>
   <property name="text">
    <string>人检2投入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
